### PR TITLE
Initial steps + first documentation HectorG 

### PR DESCRIPTION
**Pull request by HectorRute98**

### CODE REVIEW

**Comment by HectorRute98** [comment]
Add documentation to HelloController.kt and HelloControllerUnitTest.kt

File: src/main/kotlin/controller/HelloController.kt [hunk 1]
```diff
@@ -4,6 +4,9 @@ import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Controller
 import org.springframework.web.bind.annotation.GetMapping
 
+/* This class HelloController.kt creates a variable of type map and implements a function
```
Comment (fjlopez) [inline src/main/kotlin/controller/HelloController.kt:7]: Check the example at https://kotlinlang.org/docs/kotlin-doc.html and format the documentation according to the best practices.

File: src/test/kotlin/controller/HelloControllerUnitTests.kt [hunk 2]
```diff
@@ -4,12 +4,17 @@ import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 
+/*  Unit test for the class LAB1-GIT-RACE/src/main/kotlin/controller/HelloController.kt
```
Comment (fjlopez) [inline src/test/kotlin/controller/HelloControllerUnitTests.kt:7]: Same.

File: src/test/kotlin/controller/HelloControllerUnitTests.kt [hunk 3]
```diff
@@ -4,12 +4,17 @@ import org.assertj.core.api.Assertions.assertThat
...
+    which is designed using JUnit5 with a @BeforeEach label(methods that are executed before the
+    execution of each of the test methods) for create de controller. */
 class HelloControllerUnitTests {
     private lateinit var controller: HelloController
     @BeforeEach
     fun setup() {
         controller = HelloController()
     }
+    /* Unit method to test the functionality of the controller where we create values for him
```
Comment (fjlopez) [inline src/test/kotlin/controller/HelloControllerUnitTests.kt:16]: Same.

**Review by fjlopez — CHANGES_REQUESTED** [review]

**Comment by fjlopez** [comment]
Format the documentation properly. If you are using IntelliJ, toggle https://www.jetbrains.com/help/idea/working-with-code-documentation.html#toggle-rendered-view and you will see if the docs looks good or should be improved.

**Comment by HectorRute98** [comment]
Hello! I think that the changes in my fork have been made correctly and have been saved in this same pull request without the necessity to create a new one. 
If not, could you tell me how I should do it?.
Many thanks for your time.

**Review by fjlopez — APPROVED** [review]

**Comment by fjlopez** [comment]
Done.

**Pull request merged**